lib/compiler-llvm/src/abi/x86_64_systemv.rs

```diff
@@ -67,14 +81,20 @@ impl Abi for X86_64SystemV {
                 intrinsics
                     .void_ty
                     .fn_type(&param_types.collect::<Result<Vec<_>, _>>()?, false),
-                vec![],
+                vmctx_attributes
+                    .into_iter()
+                    .map(|x| (x, AttributeLoc::Param(0)))
+                    .collect(),
```

Comment:
This code seems repeated across the different match branches.
Why don't we move it up so we only write it once?

Reply:
Done.